src/common/logging.cpp

```diff
@@ -44,11 +44,16 @@ namespace logging
 
         // Sink to console
         auto stdout_sink = std::make_shared<spdlog::sinks::stdout_color_sink_mt>();
-
-        // Sink to files, creating new files at midnight
-        auto daily_sink = std::make_shared<spdlog::sinks::daily_file_sink_mt>(logFile, 0, 00, false, 0);
-
-        std::vector<spdlog::sink_ptr> sinks{ stdout_sink, daily_sink };
+        std::vector<spdlog::sink_ptr> sinks{ stdout_sink };
+
+        // Daily Sink, creating new files at midnight
+	if (appendData) {
+            sinks.push_back(std::make_shared<spdlog::sinks::daily_file_sink_mt>(logFile, 0, 00, false, 0));
+	}
+	// Basic sink, use OS tools to rotate logs
+	else {
+            sinks.push_back(std::make_shared<spdlog::sinks::basic_file_sink_mt>(logFile));
+        }
```

Comment:
https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md#general-code-guidlines-all-languages
- `4 space indent, do not use tabs for alignment.`

https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md#breakbeforebraces-allman
- `Braces should almost always be on a new line.`